### ISSUE TITLE
export 'VueSpinnersPlugin' (imported as 'VueSpinnersPlugin') was not found in 'vue3-spinners'

### ISSUE THREAD

**Issue by inmarelibero** [leonzalion/vue3-spinners#2]
hello, I have a problem during installation, I'm using Vue3, Webpack 5

if I do:
```
import { VueSpinnersPlugin } from 'vue3-spinners';
createApp(App)
  .use(VueSpinnersPlugin)
```

as stated in the doc (https://github.com/leonzalion/vue3-spinners/blob/main/readme.md), I get the following error:

```
export 'VueSpinnersPlugin' (imported as 'VueSpinnersPlugin') was not found in 'vue3-spinners' (possible exports: VueSpinner, VueSpinnerAudio, VueSpinnerBall, VueSpinnerBar, VueSpinnerBars, VueSpinnerBeat, VueSpinnerBounce, VueSpinnerBox, VueSpinnerCircle, VueSpinnerClimbingBox, VueSpinnerClip, VueSpinnerClock, VueSpinnerComment, VueSpinnerCore, VueSpinnerDot, VueSpinnerDots, VueSpinnerFacebook, VueSpinnerFade, VueSpinnerGears, VueSpinnerGrid, VueSpinnerGridPop, VueSpinnerHearts, VueSpinnerHourglass, VueSpinnerInfinity, VueSpinnerIos, VueSpinnerMoon, VueSpinnerOrbit, VueSpinnerOval, VueSpinnerPacman, VueSpinnerPie, VueSpinnerPropagate, VueSpinnerPuff, VueSpinnerPulse, VueSpinnerRadio, VueSpinnerRing, VueSpinnerRings, VueSpinnerRise, VueSpinnerRotate, VueSpinnerScale, VueSpinnerSkew, VueSpinnerSquare, VueSpinnerSync, VueSpinnerTail)
```

I tried also: `import { VueSpinnersPlugin } from 'vue3-spinners/utils/plugin';`, but got:

```
Module not found: Error: Package path ./utils/plugin is not exported from package [...]/node_modules/vue3-spinners (see exports field in [...]/node_modules/vue3-spinners/package.json)

ERROR in ./src/main.js 7:0-63
Module not found: Error: Package path ./utils/plugin is not exported from package [...]/node_modules/vue3-spinners (see exports [...]/node_modules/vue3-spinners/package.json)
```

**Comment by leonsilicon** [comment]
Should be fixed in `1.2.2`!

**Comment by inmarelibero** [comment]
@leonzalion confirmed, working now, thanks!
